js/apps/system/_admin/aardvark/APP/react/src/components/select/SelectBase.tsx

```diff
@@ -15,13 +16,23 @@ export type OptionType = {
 const Option = <IsMulti extends boolean>(
   props: OptionProps<OptionType, IsMulti>
 ) => {
+  const finalChildren =
+    typeof props.children === "boolean" ? `${props.children}` : props.children;
```

Comment:
Is this condition required? We can use `String()` to do the same.

Reply:
but what if children is not something which should be converted to a string?